outreach/metadata.go

```diff
@@ -0,0 +1,16 @@
+package outreach
```

Comment:
This can be removed because of https://github.com/amp-labs/connectors/pull/195

Reply:
Ok